samples/quickstart.js

```diff
@@ -13,31 +13,23 @@
 
 'use strict';
 
-async function main() {
+async function main(projectId) {
   // [START nodejs_service_management_quickstart]
-  // Imports the Google Cloud client library
 
-  // remove this line after package is released
-  // eslint-disable-next-line node/no-missing-require
-  const {ServiceManagerClient} = require('');
+  // Imports the Google Cloud client library
+  const {ServiceManagerClient} = require('@google-cloud/service-management');
 
   // TODO(developer): replace with your prefered project ID.
   // const projectId = 'my-project'
 
   // Creates a client
-  // eslint-disable-next-line no-unused-vars
-  const client = new {ServiceManagerClient}();
+  const client = new ServiceManagerClient();
 
-  //TODO(library generator): write the actual function you will be testing
-  async function doSomething() {
-    console.log(
-      'DPE! Change this code so that it shows how to use the library! See comments below on structure.'
-    );
-    // const [thing] = await client.methodName({
-    // });
-    // console.info(thing);
+  async function listServices() {
+    const [thing] = await client.listServices({producerProjectId: projectId});
```

Comment:
perhaps rename `thing` to `services`? and sometimes if I know the response is a list, I might do:

```js
for (const service of services) {
  console.info(service);
}
```

Reply:
Haha yes, sorry that was a vestige of the old code!